public/xmlns/page.xsd

```diff
@@ -32,10 +32,20 @@
 
         <xs:attribute name="listeners" type="content:listeners">
             <xs:annotation>
-                <xs:documentation>event_ids that trigger this page</xs:documentation>
+                <xs:documentation>This is a list of event_ids that will cause navigation to jump to this page. Page
+                    listeners are always "active".
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="dismiss-listeners" type="content:listeners">
+            <xs:annotation>
+                <xs:documentation>This is the list of event_ids that will pop this page off a navigation stack. This
+                    listener is only "active" when the page is on top of the navigation stack.
```

Comment:
Is the navigation stack, the list of pages available?

Reply:
the navigation stack is a concept for a non-linear tool type (cyoa). Each page is standalone and uses events/listeners to navigate to other pages. The dismiss listener is like a back button.

So if you start on a table of contents page, press button a which uses an event to trigger page a, then on page a there is a button that fires an event that matches the dismiss listener, it will dismiss page a and go back to the table of contents which is the next page down the page "stack"

Reply:
Thank you for explaining! I remember talk of this new tool type.